apps/AppWithWearable/lib/pages/chat/page.dart

```diff
@@ -14,6 +14,8 @@ import 'package:friend_private/backend/preferences.dart';
 import 'package:friend_private/pages/chat/widgets/ai_message.dart';
```

Comment:
This should be a dropdown that goes on the Appbar only when chat page is selected.

No bottom sheet, a dropdown.

It should contain the plugin name + plugin picture.

It should look good, and match the current design.

When there are no plugins enabled it should have that default dropdown item, and a 2nd one with enable plugins click, that opens the plugins page.

Reply:
sounds good. I appreciate these more clear deliverables. They guide development better